### PR TITLE
fix(package): Rename source package to match application id

### LINKED ISSUE #117: App's package has changed to 'io.sentry.mobile.app' but source code package is still 'io.sentrymobile.app'
Let's rename the source folder too
https://github.com/getsentry/sentry-mobile/tree/main/android/app/src/main/kotlin/io/sentrymobile/app

this would mark the App's frames as inApp=false and would affect grouping since we rely on the App's package by default

### PR DESCRIPTION
**Pull request by denrase**
Closes #117

### CODE REVIEW

File: android/app/src/main/kotlin/io/sentry/mobile/app/MainActivity.kt [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 import androidx.annotation.NonNull
 import io.flutter.embedding.android.FlutterActivity
 import io.flutter.embedding.engine.FlutterEngine
 import io.flutter.plugin.common.MethodChannel
 import android.os.AsyncTask
-import io.sentrymobile.app.NativeCrashJava
+import io.sentry.mobile.app.NativeCrashJava
 
 class MainActivity: FlutterActivity() {
     private val CHANNEL = "app.sentrymobile.io/nativeCrash"
```
Comment (marandaneto) [inline android/app/src/main/kotlin/io/sentry/mobile/app/MainActivity.kt:11]: what's about renaming this too so we match the app's package?

https://github.com/getsentry/sentry-mobile/blob/967a3f1585c492ad79e4e6228e0db684d5168e77/ios/Runner/AppDelegate.swift#L14
https://github.com/getsentry/sentry-mobile/blob/967a3f1585c492ad79e4e6228e0db684d5168e77/android/app/src/main/kotlin/io/sentrymobile/app/MainActivity.kt#L11
https://github.com/getsentry/sentry-mobile/blob/4080ac59161687e616a3505d85a2c482a59c70b3/lib/screens/debug/sentry_flutter_screen.dart#L14

**Comment by denrase** [comment]
Wait with merging until https://github.com/getsentry/sentry-java/pull/1283 is released.

**Review by marandaneto — APPROVED** [review]
LGTM

**Comment by marandaneto** [comment]
https://github.com/getsentry/sentry-dart/releases/tag/4.1.0-nullsafety.1 already unblock this, but you probably need to move to null-safety first

**Pull request merged**